networking/net_plugin.go

```diff
@@ -116,6 +119,7 @@ func (e *podEnv) execNetPlugin(cmd string, n *activeNet, netns string) ([]byte,
 	}
 
 	vars := [][2]string{
+		{"CNI_VERSION", "0.3.0"},
```

Comment:
this is not prominent enough, CNI expects a compliance version now? I guess this should be a constant, and how do we ensure compatibility with the vendored CNI version?